lms/static/js/groups/views/cohort_form.js

```diff
@@ -35,12 +35,22 @@ var edx = edx || {};
         render: function() {
             this.$el.html(this.template({
                 cohort: this.model,
+                isDefaultCohort: this.isDefault(this.model.get('name')),
                 contentGroups: this.contentGroups,
                 studioGroupConfigurationsUrl: this.context.studioGroupConfigurationsUrl
             }));
             return this;
         },
 
+        isDefault: function(name) {
```

Comment:
I still am uneasy about treating a cohort that the user may have created (as opposed to "Default Group", which we create in the code) as the default cohort. But I will let @explorerleslie decide if the current behavior is OK.